.github/ISSUE_TEMPLATE/new-content.md

```diff
@@ -4,11 +4,9 @@ about: Submit content to be reviewed and published
 title: Content's title
 labels: content
 assignees: ''
-hashtags:
-  - hash1
-  - hash2
 ---
 
 url: https://example.com/
+hashtags: #hash1,#hash2
```

Comment:
Regarding this new line, how do you think that we could manage the case when someone don't want to use any hashtag? can they just remove this line or do they need to leave it empty?

Reply:
Yep with this https://github.com/sre-paris/reliability.re/pull/37/files#diff-dc96b1335dec01e5a2533179be0dc7a142292f4c67a06390dc33a852259dfb30R66

Reply:
Yes, thanks to this 4c1d370e0ded8a0094ab41a2ba51ab9c065f92af

Would be easier to maintain by iterating over the content, and based on characters before the first `:` use the strategy pattern; `url:` then call the right strategy,...

Reply:
On twitter I added a test to check if `hashtags` is present.